NORSE_actor/hdf5Write.py

```diff
@@ -0,0 +1,60 @@
+# hdf5Write.py
+# 12/12/2019
+# Written by Soma Olasz
+#
+# This function is created to write data to hdf5 file.
+
```

Comment:
Should explore the use of standard headers, like:
# -*- coding: utf-8 -*-
"""
Created on Mon Aug 12 11:45:34 2019

@author: Fehérvári Gergő
"""

Reply:
Done for two scripts, the rest will be dealt with separately, when dealing with the relevant [issue](https://github.com/osrep/NORSE_actor/issues/16).